packages/jaeger-ui/src/components/TracePage/index.js

```diff
@@ -105,17 +110,17 @@ export class TracePageImpl extends React.PureComponent<TracePageProps, TracePage
   state: TracePageState;
 
   _headerElm: ?Element;
+  _filterSpans: typeof filterSpans;
   _searchBar: { current: Input | null };
   _scrollManager: ScrollManager;
+  traceDagEV: Object;
```

Comment:
Why make this an object instead of something with `{ edges: ..., vertices: ... }` or direct `_edges` and `_vertices`?

Reply:
if `convPlexus`'s return value was typed, then this and other places could simply import that return value. I started trying to type it but wasn't quite satisfying flow and the errors it gave were not simple to debug. Since we're about to convert to typescript, I'd rather add a todo comment.
`// TODO: typescript conversion - use convPlexus return type`